app/src/main/res/values/strings.xml

```diff
@@ -443,6 +443,10 @@
     <string name="activity_backup_drive_button_open_drive">Manage on Drive</string>
     <string name="mmol_mol">mmol/mol</string>
     <string name="mmol_L">mmol/L</string>
+    <string name="mmol_mol_value">%1$s mmol/mol</string>
```

Comment:
Do you mean the `mmol_mol_value`, `mmol_L_value` and `mg_dL_value` number strings? I added them to support localization because I thought concatenating `glucoseReading + getString(R.string.mmol_L)` would be bad practice in a left-to-right language locale.